ui/components/component-library/skeleton/skeleton.scss

```diff
@@ -0,0 +1,22 @@
+.mm-skeleton {
+  @keyframes skeleton-pulse {
+    0%,
+    100% {
+      opacity: 0.2;
+    }
+
+    50% {
+      opacity: 0.5;
+    }
+  }
+
+  animation: skeleton-pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;
+  background-clip: padding-box;
+  cursor: default;
+  pointer-events: none;
+  user-select: none;
+
+  &--hide-children * {
+    visibility: hidden;
+  }
+}
```

Comment:
were these cleared with @Akatori-Design ?

Reply:
Getting him to review also

Reply:
Good call I'm asking @Akatori-Design to review as well https://github.com/MetaMask/metamask-extension/pull/29082#issuecomment-2537265842

Reply:
Updated after syncing with @Akatori-Design on easing, timing and opacity.


https://github.com/user-attachments/assets/3528cd88-f18f-4238-ad1b-d1490cab1e93